resources/js/script.js

```diff
@@ -26,145 +28,173 @@ $(".add-to-cart").click(function(event){
 	
 	
 	
-	addItemToCart(name, price, count);
+	shoppingCart.addItemToCart(name, price, count);
 	displayCart();
 });
 
 $("#clear-cart").click(function(event){
-	clearCart();
+	shoppingCart.clearCart();
 	displayCart();
 });
 
 function displayCart() {
-	var cartArray = listCart();
+	var cartArray = shoppingCart.listCart();
 	var output = "";
+	var position = 1;
 	for (var i in cartArray) {
-		output += "<li>"+cartArray[i].name+" "+"<span>"+cartArray[i].count+"</span>"+"</li>";
+		output += "<tr>"+"<td>"+"<button class='delete-item' data-name='"+cartArray[i].name+"'>X</button>"+"<p>"+position+"</p>"+"</td>"+"<td>"+cartArray[i].name+"</td>"+"<td>"+"<span>"+cartArray[i].count+"</span>"+"<div class='add-remove-buttons'>"+" <button class='plus-item' data-name='"+cartArray[i].name+"'> + </button> "+"<button class='substract-item' data-name='"+cartArray[i].name+"'> - </button> "+"</div>"+"</td>"+"<td>"+cartArray[i].price+"</td>"+"<td>"+cartArray[i].total+"</td>"+"</tr>";
```

Comment:
I would break this into multiple lines, because on one line it's hard to read.

```javascript
output += "<tr>" + 
     "<td>" + 
             "<button class='delete-item' data-name='" + cartArray[i].name+ "'>X</button>" + 
             "<p>" + position + "</p>" + 
      "</td>" + 
      "<td>"
         + cartArray[i].name +
     "</td>"+ 
     "<td>" + 
           "<span>"+cartArray[i].count+"</span>" + 
  </td>
  .......
+"</tr>"
```

Something like this, :D

Reply:
I'll do that now :)

Reply:
Luckily for us [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) exist. They can be used to interpolate/embed values into string. So this ugly chunk of + + + + can be replaced with: 
```
output += `
  <tr>
    <td>
      <button class='delete-item' data-name='"+cartArray[i].name+"'>X</button>
      <p>${position}</p>
    </td>
    <td>
      ${cartArray[i].name}
    </td>
    <td>
      <span>${cartArray[i].count}</span>
      <div class='add-remove-buttons'>
        <button class='plus-item' data-name='${cartArray[i].name}'></button>
        <button class='substract-item' data-name='${cartArray[i].name}'> - </button>
      </div>
    </td>
    <td>
      ${cartArray[i].price}
    </td>
    <td>
      ${cartArray[i].total}
    </td>
  </tr>
`
```